gsd/hoomd.py

```diff
@@ -1092,3 +1097,44 @@ def open(name, mode='rb'):
                          schema_version=[1, 4])
 
     return HOOMDTrajectory(gsdfileobj)
+
+def read_log(name):
+    """Read scalar values from a hoomd schema GSD file.
+
+    Args:
+        name (str): File name to open.
+
+    Returns:
+        `pandas.DataFrame`
+    """
+    if fl is None:
+        raise RuntimeError("file layer module is not available")
+    if gsd is None:
+        raise RuntimeError("gsd module is not available")
+    if pandas is None:
+        raise RuntimeError("pandas module is not available")
+
+    gsdfileobj = fl.open(name=str(name),
+                         mode='rb',
+                         application='gsd.hoomd ' + gsd.__version__,
+                         schema='hoomd',
+                         schema_version=[1, 4])
+
+    logged_data_names = gsdfileobj.find_matching_chunk_names('log/')
```

Comment:
```suggestion
    logged_data_names = gsdfileobj.find_matching_chunk_names('log/')
    logged_data_names.insert(0, 'configuration/step')
```
Provide the user with the timestep associated with each log entry. HOOMD always writes `configuration/step`.